test/test_cost_model.py

```diff
@@ -22,24 +25,265 @@
 THE SOFTWARE.
 """
 
-import numpy as np
-import numpy.linalg as la  # noqa
+import pytest
+from pyopencl.tools import (  # noqa
+    pytest_generate_tests_for_pyopencl as pytest_generate_tests)
 
-from boxtree.tools import ConstantOneExpansionWrangler
 from meshmode.array_context import PyOpenCLArrayContext
+
+import numpy as np
 import pyopencl as cl
-import pyopencl.clmath  # noqa
-import pytest
-from pyopencl.tools import (  # noqa
-        pytest_generate_tests_for_pyopencl as pytest_generate_tests)
 
-from pytools import one
+from boxtree.tools import ConstantOneExpansionWrangler
+from pytential.qbx import QBXLayerPotentialSource
 from sumpy.kernel import LaplaceKernel, HelmholtzKernel
-
 from pytential import bind, sym, norm  # noqa
 from pytential import GeometryCollection
+from pytools import one
+
+from pytential.qbx.cost import (
+    QBXCostModel, _PythonQBXCostModel, make_pde_aware_translation_cost_model
+)
+
+import time
+
+import logging
+import os
+logging.basicConfig(level=os.environ.get("LOGLEVEL", "WARNING"))
```

Comment:
Please remove, as it sets the log level globally, which may be surprising.

Reply:
My intention is to allow controlling the log level from the command line. For example,
```
LOGLEVEL=INFO python test/test_cost_model.py 'test_compare_cl_and_py_cost_model(cl.create_some_context)'
```

I'm not familiar with the usual way for configuring the logging system. If we do not configure it globally here, how else can we control the log level?